File: week-07/day-01/36.js
'use strict';

var ah = [3, 4, 5, 6, 7];
// print the sum of all numbers in ah

// var i = 0
var sum = 0
// while(i < ah.length){
//   sum += ah[i]
//   i++
// }
// console.log(sum)

for(var i = 0; i < ah.length; i++){
  sum += ah[i]
}

console.log(sum)

File: week-03/day-1/36.py
ah = [3, 4, 5, 6, 7]
summa = 0
# print the sum of all numbers in ah
#s1
for n in range(len(ah)):
    summa += ah[n]
print(summa)
#s2
sumwh = 0
n = 0
while len(ah) > n:
    sumwh += ah[n]
    n += 1
print(sumwh)
#s3
print(sum(ah))
